redisinsight/desktop/src/lib/server/server.ts

```diff
@@ -1,28 +1,61 @@
 import log from 'electron-log'
 import getPort, { portNumbers } from 'get-port'
 
+import { createServer } from 'net'
 import { wrapErrorMessageSensitiveData } from 'desktopSrc/utils'
 import { configMain as config } from 'desktopSrc/config'
 
-import { AbstractWindowAuthStrategy } from 'apiSrc/modules/auth/window-auth/strategies/abstract.window.auth.strategy'
 import { getWindows } from '../window'
 
-import { WindowAuthModule } from '../../../../api/dist/src/modules/auth/window-auth/window-auth.module'
-import { WindowAuthService } from '../../../../api/dist/src/modules/auth/window-auth/window-auth.service'
-import server from '../../../../api/dist/src/main'
```

Comment:
I think we need a server for a production build.
Did you check the builds?

Reply:
you can create a branch from this branch with the name `build/....` and circleci will auto create all builds

Reply:
Tested it out. Builds work fine. I went on an insane chase of issues with dependencies between build, local and tests. Turned out that if these lines 

import { AbstractWindowAuthStrategy } from '../../../../api/dist/src/modules/auth/window-auth/strategies/abstract.window.auth.strategy'
import { WindowAuthModule } from '../../../../api/dist/src/modules/auth/window-auth/window-auth.module'
import { WindowAuthService } from '../../../../api/dist/src/modules/auth/window-auth/window-auth.service'
import server from '../../../../api/dist/src/main'

are changed to

import { AbstractWindowAuthStrategy } from 'apiSrc/modules/auth/window-auth/strategies/abstract.window.auth.strategy'
import { WindowAuthModule } from 'apiSrc/modules/auth/window-auth/window-auth.module'
import { WindowAuthService } from 'apiSrc/modules/auth/window-auth/window-auth.service'
import server from 'apiSrc/main'

Even though it is the same thing in the end, it leads to a circular dependency only in the built version